changes/26.3.3.md

```diff
@@ -1,7 +1,16 @@
 * \[**Breaking**\] Change of variant names:
```

Comment:
If it is breaking, then make the version number 27.0.0, and remove all lines in `variants.toml` containing `nonBreakingVariantAdditionPriority` or `nonBreakingTagForNewVariantSelector`.

Reply:
In addition, _measure_ the impact of glyph quantity of your change.